wiki/Beatmap_Spotlights/Leaderboards/en.md

```diff
@@ -0,0 +1,310 @@
+---
+tags:
+  - spotlights
+  - charts
+  - ranking charts
+  - spotlights leaderboard
+---
+
+# Beatmap Spotlights Test Run (Summer 2020): Ranking
+
+Here you can find the seasonal and weekly lobby leaderboards for each playlist.
```

Comment:
I think this sentence is redundant (the article is dedicated to the leaderboards), perhaps you should put a note on how frequently this is updated instead